data/autoyast_sle12/create_hdd/create_hdd_sles_regression.xml.ep

```diff
@@ -0,0 +1,411 @@
+          <type/>
+        </listentry>
+      </s390-devices>
+    </networking>
+    % }
+    % unless ($check_var->('ARCH', 's390x')) {
+    <bootloader>
+      <global>
+        <timeout config:type="integer">45</timeout>
+      </global>
+    </bootloader>
+    <general>
+      <mode>
+        % if ($check_var->('BACKEND', 'ipmi') or $check_var->('BACKEND', 'svirt')) {
+        <final_reboot config:type="boolean">true</final_reboot>
+        % }
+        <confirm config:type="boolean">false</confirm>
+      </mode>
+      <signature-handling>
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+        <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+      </signature-handling>
+    </general>
+    % if ($check_var->('ARCH', 'aarch64')) {
+    <partitioning config:type="list">
+      <drive>
+        <initialize config:type="boolean">true</initialize>
+        <use>all</use>
+        <partitions config:type="list">
+          <partition>
+            <create config:type="boolean">true</create>
+            <filesystem config:type="symbol">vfat</filesystem>
+            <format config:type="boolean">true</format>
+            <mount>/boot/efi</mount>
+            <mountby config:type="symbol">uuid</mountby>
+            <partition_id config:type="integer">259</partition_id>
+            <partition_nr config:type="integer">1</partition_nr>
+            <resize config:type="boolean">false</resize>
+            <size>300M</size>
+          </partition>
+          <partition>
+            <create config:type="boolean">true</create>
+            <create_subvolumes config:type="boolean">true</create_subvolumes>
+            <filesystem config:type="symbol">btrfs</filesystem>
+            <format config:type="boolean">true</format>
+            <mount>/</mount>
+            <mountby config:type="symbol">uuid</mountby>
+            <partition_id config:type="integer">131</partition_id>
+            <partition_nr config:type="integer">2</partition_nr>
+            <resize config:type="boolean">false</resize>
+            <size>max</size>
+          </partition>
+          <partition>
+            <create config:type="boolean">true</create>
+            <filesystem config:type="symbol">swap</filesystem>
+            <format config:type="boolean">true</format>
+            <mount>swap</mount>
+            <mountby config:type="symbol">uuid</mountby>
+            <partition_id config:type="integer">130</partition_id>
+            <partition_nr config:type="integer">3</partition_nr>
+            <resize config:type="boolean">false</resize>
+            <size>2G</size>
+          </partition>
+        </partitions>
+      </drive>
+    </partitioning>
+    % }
+    % unless ($check_var->('ARCH', 'aarch64')) {
```

Comment:
doesn´t exist normal `% else`, I found this difficult to maintain... doesn't get a proper partitioning if you just pass a few field or just the section below for aarch64?

Reply:
I have doubts how reusable is this when someone else try to use this profile for other architecture, because default and all might mean a different thing.

Reply:
Under this is not aarch64, it is unless. These conditions for partition already used in some Maintenance autoyast test, if have problems found we can maintain later I think.

For the 'all' or 'default' patterns, it may have difference between arches, cause this ticket is for aarch64, we can change the profile name to create_hdd_sles_regression_aarch64.xml.ep wdyt?  Later if one want to re-use this profile for other arches then create a new profile or update the profile to make it suitable for other arches.